plasma_framework/config.js

```diff
@@ -35,6 +35,21 @@ const development = {
             erc20: 2,
         },
     },
+    experimental: {
```

Comment:
shouldn't this copy default config and just override changed values? This may make it easier to reason about what changes in comparison to non-experimental